test/core/primitives/rle_bitset/runs_utils_test.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "primitives/rle_bitset/runs_utils.hpp"
+
+#include <gtest/gtest.h>
+
+TEST(RunsUtils, OrSuccess) {
```

Comment:
Please, provide short description of tests.

Reply:
Fixed